snapcraft/storeapi/__init__.py

```diff
@@ -278,9 +286,22 @@ def _download_snap(self, name, channel, arch, download_path,
                 name, download_path))
             return
         logger.info('Downloading {}'.format(name, download_path))
-        request = self.cpi.get(download_url, stream=True)
-        request.raise_for_status()
-        download_requests_stream(request, download_path)
+
+        # HttpAdapter cannot help here as this is a stream.
+        # LP: #1617765
+        not_downloaded = True
```

Comment:
I find it easier to have positive booleans. Use `downloaded`, and `not downloaded`.